### PR TITLE
cloud: update built-in-monitoring.md

### PR DESCRIPTION
**Pull request by shhdgit**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qqqdan — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md [hunk 1]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Average TiKV gRPC Duration | kv_get, kv_prewirite, kv_commit, and kv_pessimisticLock | The average time consumed in executing gRPC requests in all TiKV instances based on request types, including `kv_get`, `kv_prewrite`, and `kv_commit`. |
+| Average / P99 PD TSO Wait/RPC Duration | wait-avg/99, rpc-avg/99 | Wait: the average time or the 99th percentile of duration in waiting for PD to return TSO in all TiDB instances. <br/> RPC: the average time or the 99th percentile of duration from sending TSO requests to PD to receiving TSO in all TiDB instances. |
+| Average / P99 Storage Async Write Duration | avg, 99 | The average time or the 99th percentile of duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
+| Average / P99 Store Duration | avg, 99 | The average time or the 99th percentile of duration consumed in storing loop during asynchronously writing. |
+| Average / P99 Apply Duration | avg, 99 | The average time or the 99th percentile of duration consumed in applying loop during asynchronously writing. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:61]:
```suggestion
| Average / P99 Store Duration | avg, 99 | The average time or the 99th percentile of duration consumed in storing loop during asynchronous writing. |
| Average / P99 Apply Duration | avg, 99 | The average time or the 99th percentile of duration consumed in applying loop during asynchronous writing. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 2]
```diff
@@ -99,7 +73,7 @@ The following sections illustrate the metrics on the Monitoring page for Dedicat
 | TiKV Uptime | node | The runtime of each TiKV node since last restart. |
 | TiKV CPU Usage | node | The CPU usage statistics of each TiKV node. |
 | TiKV Memory Usage | node | The memory usage statistics of each TiKV node. |
-| TiKV IO MBps | node-write, node-read | The total bytes of read and write in each TiKV node. |
+| TiKV IO Bps | node-write, node-read | The total Input/Output bytes per second of read and write in each TiKV node. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:76]:
```suggestion
| TiKV IO Bps | node-write, node-read | The total input/output bytes per second of read and write in each TiKV node. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 3]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
-| Average / P99 Commit Log Duration | avg, 99 | The average time or P99 duration consumed by Raft to commit logs. |
-| Average / P99 Apply Log Duration | avg, 99 | The average time or P99 duration consumed by Raft to apply logs. |
+| Average Idle Connection Duration | avg-in-txn, avg-not-in-txn | The connection idle duration indicates the duration of a connection being idle.<br/> avg-in-txn: The average connection idle duration when a connection is within a transaction. <br/>avg-not-in-txn: The average connection idle duration when a connection is not within a transaction. |
+| Get Token Duration | avg, 99 | The average time or the 99th percentile of duration consumed in getting tokens of SQL statements. |
+| Parse Duration | avg, 99 | The average time or the 99th percentile of duration consumed in parsing SQL statements. |
+| Compile Duration | avg, 99 | The average time or the 99th percentile of duration consumed in compiling the parsed SQL AST to execution plans. |
+| Execute Duration | avg, 99 | The average time or the 99th percentile of duration consumed in executing execution plans of SQL statements. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:55]:
```suggestion
| Get Token Duration | avg, 99 | The average or the 99th percentile duration consumed in getting tokens of SQL statements. |
| Parse Duration | avg, 99 | The average or the 99th percentile duration consumed in parsing SQL statements. |
| Compile Duration | avg, 99 | The average or the 99th percentile duration consumed in compiling the parsed SQL AST to execution plans. |
| Execute Duration | avg, 99 | The average or the 99th percentile duration consumed in executing execution plans of SQL statements. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 4]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Average / P99 Storage Async Write Duration | avg, 99 | The average time or the 99th percentile of duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
+| Average / P99 Store Duration | avg, 99 | The average time or the 99th percentile of duration consumed in storing loop during asynchronously writing. |
+| Average / P99 Apply Duration | avg, 99 | The average time or the 99th percentile of duration consumed in applying loop during asynchronously writing. |
+| Average / P99 Append Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to append logs. |
+| Average / P99 Commit Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to commit logs. |
+| Average / P99 Apply Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to apply logs. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md]:
```suggestion
| Average / P99 PD TSO Wait/RPC Duration | wait-avg/99, rpc-avg/99 | Wait: the average or the 99th percentile duration in waiting for PD to return TSO in all TiDB instances. <br/> RPC: the average time or the 99th percentile of duration from sending TSO requests to PD to receiving TSO in all TiDB instances. |
| Average / P99 Storage Async Write Duration | avg, 99 | The average or the 99th percentile duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
| Average / P99 Store Duration | avg, 99 | The average or the 99th percentile duration consumed in storing loop during asynchronous writing. |
| Average / P99 Apply Duration | avg, 99 | The average or the 99th percentile duration consumed in applying loop during asynchronous writing. |
| Average / P99 Append Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to append logs. |
| Average / P99 Commit Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to commit logs. |
| Average / P99 Apply Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to apply logs. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 5]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Execute Duration | avg, 99 | The average time or the 99th percentile of duration consumed in executing execution plans of SQL statements. |
+| Average TiDB KV Request Duration | Get, Prewirite, Commit, and PessimisticLock | The average time consumed in executing KV requests in all TiDB instances based on request types, including `Get`, `Prewrite`, and `Commit`. |
+| Average TiKV gRPC Duration | kv_get, kv_prewirite, kv_commit, and kv_pessimisticLock | The average time consumed in executing gRPC requests in all TiKV instances based on request types, including `kv_get`, `kv_prewrite`, and `kv_commit`. |
+| Average / P99 PD TSO Wait/RPC Duration | wait-avg/99, rpc-avg/99 | Wait: the average time or the 99th percentile of duration in waiting for PD to return TSO in all TiDB instances. <br/> RPC: the average time or the 99th percentile of duration from sending TSO requests to PD to receiving TSO in all TiDB instances. |
+| Average / P99 Storage Async Write Duration | avg, 99 | The average time or the 99th percentile of duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:59]:
```suggestion
| Average / P99 PD TSO Wait/RPC Duration | wait-avg/99, rpc-avg/99 | Wait: the average time or the 99th percentile of duration in waiting for PD to return TSO in all TiDB instances. <br/> RPC: the average time or the 99th percentile of duration from sending TSO requests to PD to receiving TSO in all TiDB instances. |
| Average / P99 Storage Async Write Duration | avg, 99 | The average time or the 99th percentile of duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
```

File: tidb-cloud/built-in-monitoring.md [hunk 6]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Average / P99 Store Duration | avg, 99 | The average time or the 99th percentile of duration consumed in storing loop during asynchronous writing. |
+| Average / P99 Apply Duration | avg, 99 | The average time or the 99th percentile of duration consumed in applying loop during asynchronous writing. |
+| Average / P99 Append Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to append logs. |
+| Average / P99 Commit Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to commit logs. |
+| Average / P99 Apply Log Duration | avg, 99 | The average time or the 99th percentile of duration consumed by Raft to apply logs. |
```
Comment (qiancai) [inline tidb-cloud/built-in-monitoring.md:64]:
```suggestion
| Average / P99 PD TSO Wait/RPC Duration | wait-avg/99, rpc-avg/99 | Wait: the aaverage or the 99th percentile duration in waiting for PD to return TSO in all TiDB instances. <br/> RPC: the average time or the 99th percentile of duration from sending TSO requests to PD to receiving TSO in all TiDB instances. |
| Average / P99 Storage Async Write Duration | avg, 99 | The average or the 99th percentile duration consumed in asynchronous writing. Average storage async write duration = Average store duration + Average apply duration. |
| Average / P99 Store Duration | avg, 99 | The average or the 99th percentile duration consumed in storing loop during asynchronous writing. |
| Average / P99 Apply Duration | avg, 99 | The average or the 99th percentile duration consumed in applying loop during asynchronous writing. |
| Average / P99 Append Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to append logs. |
| Average / P99 Commit Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to commit logs. |
| Average / P99 Apply Log Duration | avg, 99 | The average or the 99th percentile duration consumed by Raft to apply logs. |
```

**Review by qiancai — APPROVED** [review]

File: tidb-cloud/built-in-monitoring.md [hunk 7]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Average Idle Connection Duration | avg-in-txn, avg-not-in-txn | The connection idle duration indicates the duration of a connection being idle.<br/> avg-in-txn: The average connection idle duration when a connection is within a transaction. <br/>avg-not-in-txn: The average connection idle duration when a connection is not within a transaction. |
+| Get Token Duration | avg, 99 | The average or the 99th percentile duration consumed in getting tokens of SQL statements. |
+| Parse Duration | avg, 99 | The average or the 99th percentile duration consumed in parsing SQL statements. |
+| Compile Duration | avg, 99 | The average or the 99th percentile duration consumed in compiling the parsed SQL AST to execution plans. |
+| Execute Duration | avg, 99 | The average or the 99th percentile duration consumed in executing execution plans of SQL statements. |
+| Average TiDB KV Request Duration | Get, Prewirite, Commit, and PessimisticLock | The average time consumed in executing KV requests in all TiDB instances based on request types, including `Get`, `Prewrite`, and `Commit`. |
```
Comment (ran-huang) [inline tidb-cloud/built-in-monitoring.md:56]: label 里的 `PessimisticLock` 没有 description，需要加上吗？

File: tidb-cloud/built-in-monitoring.md [hunk 8]
```diff
@@ -29,65 +29,39 @@ To view the metrics on the Monitoring page, take the following steps:
...
+| Average TiKV gRPC Duration | kv_get, kv_prewirite, kv_commit, and kv_pessimisticLock | The average time consumed in executing gRPC requests in all TiKV instances based on request types, including `kv_get`, `kv_prewrite`, and `kv_commit`. |
```
Comment (ran-huang) [inline tidb-cloud/built-in-monitoring.md:57]: 同上

**Review by ran-huang — APPROVED** [review]
rest LGTM

File: tidb-cloud/built-in-monitoring.md (hunk 7 above)
Comment (shhdgit) [reply tidb-cloud/built-in-monitoring.md]: 目前多了很多 type，我先改成这样了，PTAL。https://github.com/pingcap/docs/pull/13333/commits/3499f46175601bbbf1a28decefc3171861772c2e

**Review by ran-huang — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**